bin/analyze_hoc_activity

```diff
@@ -150,6 +150,23 @@ def main()
     day += 1
   end
 
+  hoc_country_totals = {}
+  hoc_company_totals = {}
+  unique_hoc_events = DB[:forms].where(kind: 'HocSignup2015').group(:name, :email).select(:name, :email, :processed_data, :data)
+  total_hoc_count = unique_hoc_events.count
+
+  unique_hoc_events.each do |row|
+    data = JSON.parse(row[:data])
```

Comment:
I suppose any invalid json is our code bug so it's a feature that this blows up if the data is invalid; just confirming.